third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -41,28 +41,54 @@ IS_BAZEL = not (hasattr(native, "genmpm"))
 
 # Kotlin Compiler Options
 KOTLINC_OPTS = [
+    "-Xallow-kotlin-package",
+    "-Xinline-classes",
     "-Xmulti-platform",
     "-Xuse-experimental=kotlin.ExperimentalMultiplatform",
 ]
 
+DISABLED_LINT_CHECKS = [
+    "PackageName",
+    "TopLevelName",
+]
+
+def _merge_lists(*lists):
+    result = {}
+    for l in lists:
+        for elem in l:
+            result[elem] = 1
+    return result.keys()
+
 def arcs_kt_jvm_library(**kwargs):
-    """Declares arbitrary kotlin library targets for the JVM.
+    """Wrapper around kt_jvm_library for Arcs.
+
+    Provides a bunch of default arguments.
```

Comment:
This line doesn't really add anything helpful. It'd be nice to know what arguments it provides defaults for.

Reply:
I'm gonna delete it. It's annoying to keep up to date.